module/platform/Webots/src/Webots.cpp

```diff
@@ -563,17 +564,17 @@ namespace module::platform {
         if (sensor_measurements.accelerometers.size() > 0) {
             // .accelerometers is a list of one, since our robots have only one accelerometer
             const auto& accelerometer    = sensor_measurements.accelerometers[0];
-            sensor_data->accelerometer.x = static_cast<float>(accelerometer.value.X);
-            sensor_data->accelerometer.y = static_cast<float>(accelerometer.value.Y);
-            sensor_data->accelerometer.z = static_cast<float>(accelerometer.value.Z);
+            sensor_data->accelerometer.x = Convert::accelerometer(static_cast<float>(accelerometer.value.X));
+            sensor_data->accelerometer.y = Convert::accelerometer(static_cast<float>(accelerometer.value.Y));
+            sensor_data->accelerometer.z = Convert::accelerometer(static_cast<float>(accelerometer.value.Z));
```

Comment:
Should already be in SI units, just need to offset the readings
```suggestion
            sensor_data->accelerometer.x = static_cast<float>(accelerometer.value.X) - 100.0f;
            sensor_data->accelerometer.y = static_cast<float>(accelerometer.value.Y) - 100.0f;
            sensor_data->accelerometer.z = static_cast<float>(accelerometer.value.Z) - 100.0f;
```